### ISSUE TITLE
Apply Does Nothing

### ISSUE THREAD

**Issue by PolarPeridot** [togenyan/Yo-kai_Editor_3#48]
So, I've been trying to edit my save to give myself some more items, and there's a BIG problem. Whenever I hit Apply on the latest version, nothing happens. No changes commit. No matter what I do, it's not doing it. Please fix this!

**Comment by Fillerfx** [comment]
This is true for me too, neither applying a new item/yokai or modifying an existing one works.

**Comment by togenyan** [comment]
I think the cause is same as https://github.com/togenyan/Yo-kai_Editor_B2/commit/a2a7520f28252a53f8d206c2fe3495eca4622563; I'll fix it soon.
